packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -311,7 +311,7 @@ const Cart = () => {
         return <CartSkeleton />
     }
 
-    if (!isLoading && !basket?.productItems?.length) {
+    if (!basket?.productItems?.length) {
```

Comment:
If we check loading here, we won't need to guard basket below

Reply:
sounds good, removed the guard

Reply:
@alexvuong @kevinxh FYI it looks the change broke the cart page when the cart is empty.
http://localhost:3000/global/en-GB/cart

```
TypeError: Cannot read properties of undefined (reading 'length')
    at Cart (http://localhost:3000/mobify/bundle/development/pages-cart.js:394:28)
```

Reply:
ok... i'm just gonna revert the change in cart.js.... it's out of scope for this PR and i thought it's a small diff improvement. but turns out to be too much hassle